### ISSUE TITLE
Broken (reply) message link when repling with an URI

### ISSUE THREAD

**Issue by invalid-error** [oxen-io/session-android#704]
- [x] I have read and agree to adhere to the [Code of Conduct]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Loosing referred message (reply to message) when replying with an URI.

### Steps to reproduce
- Settings > privacy > set 'Send link previews' active
- go to a closed or open group and reply to a message (left swipe or click-hold and reply to message)
- write some text and add a URI (i.e. https://getsession.org/fag/)
- 'x' - remove link preview; send message.

Describe here what happens after you run the steps above (i.e. the buggy behaviour)
- your reply doesn't link (reply) to the original message.
- removing the link preview also removes the link to the message you reply to.

**Session version:** 1.11.8

**Comment by eyeberic** [comment]
Might be related to #664 .

**Comment by Marky-N** [comment]
Is possibly related to iOS [#494](https://github.com/oxen-io/session-ios/issues/494)

**Comment by KeeJef** [comment]
is this fixed with the above patch? @invalid-error

**Comment by KeeJef** [comment]
Closing as I'm assuming this has been fixed

**Comment by Marky-N** [comment]
This has not been fixed, at least not for iOS.

**Comment by Marky-N** [comment]
@KeeJef Please open this issue, as this is NOT fixed!

**Comment by KeeJef** [comment]
Yes it seems this is not resolved on Android, i can reproduce this

**Comment by Marky-N** [comment]
@KeeJef As written above, same issue applies to iOS as well.

**Comment by KeeJef** [comment]
Please open an issue on the iOS repo if there isn't one open already 

> @KeeJef As written above, same issue applies to iOS as well.
